tests/test_repair.py

```diff
@@ -318,29 +318,30 @@ def test__grow():
     obj._grow(leaf, 0, obj.neuron.soma.center)
     assert_equal(len(obj.neuron.sections), 8)
 
+
 def test_repair_axon():
     filename = DATA_PATH / 'real-with-axon.asc'
     with TemporaryDirectory('test-cli-axon') as tmp_folder:
         outfilename = Path(tmp_folder, 'out.asc')
         test_module.repair(filename, outfilename, axons=[filename])
         neuron_in = load_neuron(filename)
         neuron_out = load_neuron(outfilename)
-        axon = neuron_out.section(40)
+        axon = neuron_out.section(41)
```

Comment:
so, MorphIO counts soma as a section? I thought it starts section indexing from 0. To be honest I expected to see here 39 instead.

Reply:
because this axon has a unifurcation, which is not merged now, so it has one more section. I think the unifurcation was before this section. It took me a while to figure it out haha